docs/integrations/databases/firebase.mdx

```diff
@@ -5,163 +5,176 @@ description: Learn how to integrate Clerk into your Firebase application.
 
-To upload the file to Clerk, click on the **Upload JSON** button on the Firebase integration screen inside the configuration modal.
+To get started:
+1. Navigate to the Clerk Dashboard.
+1. In the sidebar, select **[Integrations](https://dashboard.clerk.com/last-active?path=integrations)**, and toggle the Firebase integration on.
 
-<Images
-  width={3024}
-  height={1652}
-  src="/docs/images/integrations/firebase/upload-json.webp"
-  alt="The Firebase configuration form in the Clerk Dashboard. A red arrow is pointing to the 'Upload JSON' button."
-/>
+The Firebase integration enables you to use Clerk to generate a valid authentication token to send to Firebase's API. To do so, you can upload your Firebase project's service account key JSON file to configure the integration automatically.
 
-### Fill in the required attributes manually
+To generate a service account key:
 
-The attributes that you need to fill in to connect Clerk with Firebase, as a custom authentication system, are described below.
+1. Visit [the Service Accounts settings](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) in your Firebase project.
+1. Near the bottom of the page, select **Generate New Private Key** and save the file.
 
-- **Service account ID** - The **service account ID** can be found in the [Google Cloud Console](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts?consoleUI=FIREBASE&hl=fi&supportedpurview=project) for your Firebase project, or in the `client_email` field of a service account JSON file.
-- **Project ID** - The **Firebase project ID** is the unique identifier of your Firebase project. Can be found under the **Project Settings** in the Firebase platform.
-- **Database URL** *(Optional)* - The **Firebase Realtime Database URL** as retrieved from the Firebase platform under the **Realtime Database** page.
-- **Private Key** - The private key used for signing which belongs to the Google service account ID of your project. Can be found in the Google Cloud Console for your Firebase project, or in the `private_key` field of a service account JSON file.
+See [the manual configuration instructions](#configure-firebase-manually) if you don't want to use a service account key.
 
-## Firebase user sign-in with Clerk as an authentication provider
+### Enable authentication in Firebase
 
-After successfully completing the integration setup on in the Clerk Dashboard, you should set up your frontend to connect Clerk with the Firebase authentication system.
+To use Firebase auth, ensure authentication is enabled in your Firebase dashboard. To do so:
 
-<Steps>
+1. Navigate to your Firebase dashboard.
+1. In the sidebar navigation, expand **Build** and select **Authentication**.
+1. Enable authentication on this page, and enable any sign-in method you want, such as **Email/Password**.
 
-### Retrieve a Firebase user authentication token from Clerk
+### Add a Security Rule to your Firestore database
```

Comment:
Do all users need to do this step?

```suggestion
### Add a Security Rule to your Firestore database (optional)
```

Reply:
okay just got to the last step, and I see that the example includes firestore. so we should make it clear in this section that for the sake of this tutorial, the user will need to setup firestore.
I've never set up in the firebase dashboard or used it in a project, so i'm not sure what that entails.